services/github_service.py

```diff
@@ -1129,10 +1130,21 @@ def get_current_month_gha_minutes_for_enterprise(self, month) -> int:
             "X-GitHub-Api-Version": "2022-11-28"
         }
 
-        response = self.github_client_rest_api.get(
-            f"https://api.github.com/enterprises/{self.enterprise_name}/settings/billing/usage?month={month}", headers=headers)
+        max_attempts = 3
+        for attempt in range(1, max_attempts + 1):
+            response = self.github_client_rest_api.get(
+                f"https://api.github.com/enterprises/{self.enterprise_name}/settings/billing/usage?month={month}",
+                headers=headers
+            )
 
-        return response.json()
+            data = response.json()
+            if data and data.get("usageItems") is not None:
```

Comment:
```suggestion
            if data and data.get("usageItems"):
```
is the `is not None` bit necessary? If `data.get("usageItems")` is `None` won't the `and` fail anyway?

Reply:
Indeed, thanks!